src/jit_compiler_x86_static.asm

```diff
@@ -78,14 +78,15 @@ randomx_program_prologue PROC
 	and eax, RANDOMX_SCRATCHPAD_MASK
 	ror rdx, 32
 	and edx, RANDOMX_SCRATCHPAD_MASK
-	jmp randomx_program_loop_begin
+	jmp rx_program_loop_begin
 randomx_program_prologue ENDP
 
 ALIGN 64
 	include asm/program_xmm_constants.inc
 
 ALIGN 64
 randomx_program_loop_begin PROC
+rx_program_loop_begin:
```

Comment:
Should be a double colon here, otherwise the label is scoped to the procedure and the jmp instruction won't build.

Reply:
ok.